ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -353,15 +353,24 @@
                                    (.setTimeout js/window (fn [] (scroll/scroll-to-id "route-calendar-anchor")) 150)))
                    :row-selected? #(= (:route-hash-id %) (:route-hash-id selected-route))}
 
-      [{:name "Reitti" :width "20%"
+      [{:name ""
+        :read identity
+        :format (fn [{:keys [recent-change? change-detected]}]
+                  (when recent-change?
+                    [:div {:style (stylefy/use-style style-base/new-change-container)
+                           :title (str "Muutos tunnistettu: " (time/format-timestamp->date-for-ui change-detected))}
+                     [:div (stylefy/use-style style-base/new-change-indicator)]]))
+        :col-style style-base/table-col-style-wrap
+        :width "2%"}
+       {:name "Reitti" :width "20%"
         :read (juxt :route-short-name :route-long-name)
         :col-style style-base/table-col-style-wrap
         :format (fn [[short long]]
                   (str short " " long))}
 
        ;; Show Reitti/Määränpää column only if it does affect on routes.
        (when (service-is-using-headsign route-hash-id-type)
-         {:name "Reitti/määränpää" :width "23%"
+         {:name "Reitti/määränpää" :width "21%"
```

Comment:
Just FYI, in this `service-is-using-headsign` case the column gets cramped but I guess this is the only one we can steal some screen real-estate.